ScoutSuite/providers/aws/services/sqs.py

```diff
@@ -24,10 +24,11 @@ def parse_queue(self, global_params, region, queue_url):
         :param queue_url:               URL of the AWS queue
         """
         queue = {'QueueUrl': queue_url}
-        attributes = api_clients[region].get_queue_attributes(QueueUrl = queue_url, AttributeNames = ['CreatedTimestamp', 'Policy', 'QueueArn'])['Attributes']
+        attributes = api_clients[region].get_queue_attributes(QueueUrl = queue_url, AttributeNames = ['CreatedTimestamp', 'Policy', 'QueueArn', 'KmsMasterKeyId'])['Attributes']
         queue['arn'] = attributes.pop('QueueArn')
-        for k in ['CreatedTimestamp']:
-            queue[k] = attributes[k] if k in attributes else None
+        queue['kms_master_key_id'] = attributes.pop('KmsMasterKeyId', None)
+        queue['CreatedTimestamp'] = attributes.pop('CreatedTimestamp', None)
```

Comment:
It doesn't seem consistant that `kms_master_key_id` is snake cased while `CreatedTimestamp` is camel cased.

Reply:
Yea, I hesitated on this one. I usually try to use snake case, so I decided to be consistent with that. I guess I should have been consistent with this level of the data structure. I'm not sure what to be loyal to these days.

Reply:
I try to use snake case everywhere, as that's the python standard (for most cases at least!).

> I'm not sure what to be loyal to these days.

word